packages/bitfield/src/lib/BitField.ts

```diff
@@ -1,8 +1,11 @@
+const FlagEntriesSymbol = Symbol('@sapphire/bitfield:bitfield.flags.entries');
```

Comment:
no need to complicate this so much since its Symbol, not Symbol.for

```suggestion
const FlagEntriesSymbol = Symbol('@sapphire/bitfield.flags.entries');
```